### ISSUE TITLE
[Hinode] Add underline to link hover

### ISSUE THREAD

**Issue by timcappalli** [passkeydeveloper/passkeys.dev#413]
Currently links just go bold on hover. This fails WCAG accessibility checks. Change to underline.
